### PR TITLE
A Small class that contains the possible status from the Job server

### PR DESCRIPTION
**Pull request by jomach**
Just a small helper if we want to compare status without stringing everything

### CODE REVIEW

File: sjsclient/job.py [hunk 1]
```diff
@@ -3,6 +3,12 @@
 from sjsclient import base
 from sjsclient import utils
 
+class JobStatus(object):
+    """A Helper class that contains the possible status returned by job.get().status    """
+    RUNNING = "RUNNING"
+    FINISHED = "FINISHED"
+    ERROR = "ERROR"
+
```
Comment (noorul) [inline sjsclient/job.py]: Where are we using this?

**Comment by jomach** [comment]
Is just a helper class for people using the sjs. 

for example im my case I want to check when the job is done so untill now I have done : 

if job.get("id").status == "RUNNING": 

with this I can do : 
if job.get("id").stauts == JobStatus.RUNNING

**Comment by noorul** [comment]
@jomach Thank you! Can you please fix pep8 error https://travis-ci.org/spark-jobserver/python-sjsclient/jobs/146430950

**Comment by jomach** [comment]
now.

**Comment by noorul** [comment]
Let us squash the commits into one.

**Pull request closed without merging**